platform/pocketbook/koreader.app

```diff
@@ -15,7 +15,7 @@ export KO_PATH_OPEN_BOOK="/tmp/.koreader.open"
 INSTANCE_PID=$(cat /tmp/koreader.pid 2>/dev/null)
 if [ "${INSTANCE_PID}" != "" ] && [ -e "/proc/${INSTANCE_PID}" ]; then
     echo "$@" >"${KO_PATH_OPEN_BOOK}"
-    exec /ebrmain/bin/iv2sh SetActiveTask "${INSTANCE_PID}" 0
+    exec $(if [ -f "/usr/bin/iv2sh" ]; then echo /usr/bin/iv2sh; else /ebrmain/bin/iv2sh; fi) SetActiveTask "${INSTANCE_PID}" 0
```

Comment:
Missing an echo in the else branch?

Reply:
Indeed, already fixed

Reply:
Yep, and I had already resolved the conversation. ;)